components/avatar/avatar.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Avatar-Badge</title>
+    <link rel="stylesheet" href="../../main.css">
+</head>
+<body>
+    <nav class="container">
+        <div id="logo">Icarus-UI</div>
+        <div class="a">
+            <a href="#" class="a1">Home</a>
+            <a  href="#"  class="a1">Docs</a>
+        </div>
+    </nav>
+    <div class="grid">  
+    <aside class="side-bar">
+        <div><a href="#">Installation</a></div>
+        <div><a href="#">Typography</a></div>
+        <h2>Components</h2>
+        <div><a href="#">Alert</a></div>
+        <div><a href="#">Avatar</a></div>
+        <div><a href="#">Badge</a></div>
+        <div><a href="#">Button</a></div>
+        <div><a href="#">Card</a></div>
+        <div><a href="#">Grid</a></div>
+        <div><a href="#">Image</a></div>
+        <div><a href="#">Input</a></div>
+        <div><a href="#">Toast</a></div>
+    </aside>
+    <div class="main">
+    <h1 >Avatar</h1>
+    <p class="size3">Avatar represents a image in a circular Container</p>
+    <img class="avatar" src="https://miro.medium.com/max/1193/1*NVU5-vgxMwufQAsXfr67-Q@2x.jpeg" alt="Pic">
+    <br>
+    <iframe
+    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=31px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=118%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar%2522%2520src%253D%2522https%253A%252F%252Fmiro.medium.com%252Fmax%252F1193%252F1*NVU5-vgxMwufQAsXfr67-Q%25402x.jpeg%2522%2520alt%253D%2522Pic%2522%253E%250A"
+    style="width: 915px; height: 90px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
avoid using inline styles. you can move this to your css file and then use the class.